react/AsidedLayout/AsidedLayout.js

```diff
@@ -0,0 +1,30 @@
+import styles from './AsidedLayout.less';
+import React from 'react';
+import classnames from 'classnames';
+
+const defaultRenderAside = () => null;
+
+export default function AsidedLayout({ className, children, renderAside = defaultRenderAside, size, reverse }) {
+  return (
+    <div
+      className={classnames({
+        [className]: className,
+        [styles.root]: true,
+        [styles.reverse]: reverse
+      })}>
+      { reverse ?
```

Comment:
@mjt01 doing it in the markup means you're presenting content to screen readers in the same order as the mobile view, which is much better IMO.